lib/elastic_record/index.rb

```diff
@@ -89,7 +89,7 @@ def real_connection
 
     def get(end_path, json = nil)
       path = "/#{alias_name}"
-      path += "/#{mapping_type}"
+      path += "/#{mapping_type}" if custom_mapping_type_name?
```

Comment:
Preferably, `custom_mapping_type_name?` is deleted from the code

Reply:
<img src="https://media1.tenor.com/images/edbdfe8e7dd32f6988e7de2c499052d7/tenor.gif?itemid=8420673" />